src/Agent/NewRelic/Agent/Core/OpenTelemetryBridge/MeterFilterService.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Linq;
+using NewRelic.Agent.Configuration;
+
+namespace NewRelic.Agent.Core.OpenTelemetryBridge
+{
+    public class MeterFilterService
```

Comment:
Does this class need to inherit `ConfigurationBasedService` so it can respond to configuration changes?